src/main/java/com/bytecoders/pharmaid/service/SharedPermissionService.java

```diff
@@ -19,6 +19,8 @@
 @Service
 public class SharedPermissionService {
 
+  private static final int FIRST_RESPONDER = 1;
+
```

Comment:
We should move all enums to their own class so we can reference them in a modular format

We currently have duplicate enums in multiple classes + testing